### ISSUE TITLE
the floating toolbar humps around when selecting multiple lines of text

### ISSUE THREAD

**Issue by thenanyu** [outline/rich-markdown-editor#197]
![image](https://user-images.githubusercontent.com/427579/82845561-e6f19b80-9e99-11ea-8e2d-2a6e96d05ba8.png)

This is a minor display quibble, but since the floating toolbar remains fixed at the top of a selection it should have its position based on the top line of the selection. I think what's happening is that it's recalculating based on the highlight length of the current cursor position.

**Comment by tommoor** [comment]
Makes sense, measuring just the width of the top of the selection might be quite difficult though – an approach doesn't immediately come to mind for how to achieve that, especially considering it could be any type of node and not just a paragraph.

**Comment by tommoor** [comment]
Two improvements:

- [ ] Don't show the toolbar until mouse up
- [ ] Show the toolbar at the bottom of the selection if dragging from top -> bottom

**Comment by rblalock** [comment]
Is there a reason the toolbar doesn't have a class name associated with it?  Some way to target and style it separately?

**Comment by rblalock** [comment]
Just added a PR for the className.  https://github.com/outline/rich-markdown-editor/pull/283
